### DISCUSSION TITLE
WriteLine outputs Liquid object names instead of values

### DISCUSSION THREAD

**Discussion by aaronfhamilton** [elsa-workflows/elsa-core#3502] [Q&A]
When using the WriteLine workflow activity, how does one properly use Liquid syntax to show values? I'm using the following code:

```
Input: {{ Input }}
Variables: {{ Variables }}
Message: {{ Variables.Message }}
Context ID: {{ ContextId }}
Workflow Context: {{ WorkflowContext }}
```
... but the journal output is showing this ...

```
Input: 
Variables: Elsa.Scripting.Liquid.Helpers.LiquidPropertyAccessor
Message: 
Context ID: LiquidActivityModel { ActivityExecutionContext = Elsa.Services.Models.ActivityExecutionContext, ActivityName = ContextId, ActivityId =  }
Workflow Context: LiquidActivityModel { ActivityExecutionContext = Elsa.Services.Models.ActivityExecutionContext, ActivityName = WorkflowContext, ActivityId =  }

```

**Comment by taurisjoe** [comment]
Not sure if I'm having the same difficulty or not but it appears that Activity outputs are just not available in Liquid, only via javascript.  It's frustrating because I have to create a discreet variable of an activity output if I want to use Liquid.
